maxim/tests/quick_retry_test.py

```
@@ -0,0 +1,79 @@
+"""
+Quick integration test for connection retry functionality.
+"""
+
+import sys
+
+sys.path.append("..")
+
+from maxim.apis.maxim_apis import MaximAPI
+from unittest.mock import Mock, patch
+from http.client import RemoteDisconnected
+from requests.exceptions import ConnectionError
```

Pylint (3.3.7)

[warning] Redefining built-in 'ConnectionError'

(W0622)